### PR TITLE
[MIG] Migrate hr_simplify to V8 and new API

### PR DESCRIPTION
**Pull request by feketemihai**
I made a few comments, waiting for your oppinions about it.

### CODE REVIEW

File: hr_simplify/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        return res
+
+    contract_id = fields.Many2one("hr.contract", string="Contract",
+                                  compute="_get_latest_contract",
+                                  store=True,
+                                  help="Latest contract of the employee")
+    job_id = fields.Many2one("hr.job", string="Job",
+                             related="contract_id.job_id")
+
+    _sql_constraints = [
```
Comment (feketemihai) [inline hr_simplify/models/hr_employee.py:54]: Do we need this constrain, basically if we put the constrain on this, we should put on ssnid and sinid fields which are similar.

File: hr_simplify/models/hr_job.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# -*- coding:utf-8 -*-
```
Comment (feketemihai) [inline hr_simplify/models/hr_job.py:1]: File not imported yet...help requested.

File: hr_simplify/models/hr_job.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        'no_of_employee': fields.function(
+            _no_of_contracts,
+            string="Current Number of Employees",
+            help='Number of employees currently occupying this job position.',
+            store={
+                'hr.contract': (_get_job_position, ['job_id'], 10),
+            },
+            multi='no_of_employee',
+        ),
+        'expected_employees': fields.function(
```
Comment (feketemihai) [inline hr_simplify/models/hr_job.py:62]: Basically this is increasing the number of forecacsted employees, isn't this supposed to modify no_of_recruitment field, by making a difference between expected_employees and no_of_employee fields.

**Comment by ghost** [comment]
@feketemihai My opinion is that this module should not be ported to v8. It does many things that are not related one to another. Globally, this module is incoherent. Fields in models are rewritten (e.g. state in contract model). The name of the module is also incoherent, because this module does not simplify things, it complicates them. For this reason, I would suggest to split the module in more modules and give them proper names.

**Comment by feketemihai** [comment]
@dufresnedavid I started to port the hr_contract_init to V8 and i saw that the module depends on the 'hr_job_categories', 'hr_security', 'hr_simplify', that's why i started to port the dependencies first and i didn't saw any issue regarding this modules...
Waiting for others oppinion about this...

**Comment by feketemihai** [comment]
@OCA/human-resources-maintainers After thinking more we should remove this module from the repo, it's just a mix of usability, not adding a precise feature

Thoughts?

**Comment by dreispt** [comment]
I agree on removal from 8.0 (and 9.0?). Maybe we should also add some info on the 7.0 description stating that it is deprecated.

**Comment by andhit-r** [comment]
@feketemihai @dreispt what is the final decision regarding this module? deprecated from 8.0?

**Comment by feketemihai** [comment]
From my point of view, yes...we will have to add this in the name of the module...

**Comment by andhit-r** [comment]
:+1:

**Pull request closed without merging**